daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/unparsers/runtime1/SpecifiedLengthUnparsers.scala

```diff
@@ -200,7 +124,7 @@ trait CalculatedPrefixedLengthUnparserMixin {
         UnparseError(
           One(state.schemaFileLocation),
           One(state.currentLocation),
-          s"The calculated value of ${elem.namedQName} ($adjustedLenInUnits) failed check due to ${check.errMsg}"
+          s"The calculated value of ${plElem.namedQName} ($adjustedLenInUnits) failed check due to ${check.errMsg}"
```

Comment:
Is switching to `plElem` correct? Although the check is done against `plElem`, `plElem` is really an ephemeral quasi-element that doesn't exist in the real infoset, so it might be confusing to mention it in an error message. Maybe this instead wants to be something like "The calculated prefix length of ${elem.namedQName} ..."?